docs/toc.md

```diff
@@ -189,6 +189,14 @@
 ### [dotnet-nuget-delete](core/preview3/tools/dotnet-nuget-delete.md)
 ### [dotnet-nuget-push](core/preview3/tools/dotnet-nuget-push.md)
 ### [dotnet-nuget-locals](core/preview3/tools/dotnet-nuget-locals.md)
+### Project modification commands
+#### References
+##### [dotnet-add-reference.md](core/preview3/tools/dotnet-add-reference.md)
```

Comment:
let's add these files to their final destination since preview3 location will cease to exist soon and so we avoid more redirections.